doc/classes/TextEdit.xml

```diff
@@ -926,6 +1036,9 @@
 			If [code]true[/code], a right-click moves the caret at the mouse position before displaying the context menu.
 			If [code]false[/code], the context menu disregards mouse location.
 		</member>
+		<member name="caret_multiple" type="bool" setter="set_multiple_carets_enabled" getter="is_multiple_carets_enabled" default="true">
```

Comment:
This property's name is a bit odd, but I guess not much can be done if it needs to start with `caret`. Better name would be `multiple_carets_enabled`.

Reply:
Yeah it's not the best name, but indeed has to start with `caret`.

Reply:
It doesn't actually _have to_ start with `caret`, grouping in the inspector can work fine even if the prefix doesn't match - it will just not strip the prefix so it would show:

```
Caret
- Multiple Carets Enabled
- Type
```

The property name and the setter/getter should also be kept in sync so one of the two should be renamed either way.

Could also be `multicaret_enabled` maybe.